service/remote.go

```diff
@@ -98,13 +98,13 @@ func (r *RemoteService) remoteProcess(
 	switch msg.Type {
 	case message.Request:
 		if err != nil {
-			logger.Log.Errorf("Failed to process remote: %s", err.Error())
+			logger.Log.Errorf("Failed to process remote server with id '%s' and hostname '%s': %s", server.ID, server.Hostname, err.Error())
 			a.AnswerWithError(ctx, msg.ID, err)
 			return
 		}
 		err := a.Session.ResponseMID(ctx, msg.ID, res.Data)
 		if err != nil {
-			logger.Log.Errorf("Failed to respond remote: %s", err.Error())
+			logger.Log.Errorf("Failed to respond to remote server with id '%s' and hostname '%s': %s", server.ID, server.Hostname, err.Error())
```

Comment:
Just a notice, since we're using the server id on the message, it will become hard to grep for the specific message. Can't we add these as parameters of the log instead of the message?

Reply:
I tried to add fields to the logger but the pitaya Log interface does not have fields. Do you know how to add them?

Reply:
Oh, so maybe there is none. I never added them. Maybe we should think about adding them to the interface them.